tuf/client_rework/metadata_bundle.py

```diff
@@ -142,23 +125,16 @@ class MetadataBundle(abc.Mapping):
     update the metadata with the caller making decisions on what is updated.
     """
 
-    def __init__(self, repository_path: str):
-        """Initialize by loading root metadata from disk"""
-        self._path = repository_path
+    def __init__(self, data: bytes):
```

Comment:
I would suggest renaming `data` to `root_data` to make it even more descriptive.